src/config.js

```diff
@@ -55,5 +55,5 @@ module.exports = {
 
   delimeter: '\n\r',
 
-  poiMarker: '◉',
-};
+  poiMarker: '◉'
+}
```

Comment:
Removing trailing semicolons deviates from the current code style.
I'd prefer to configure the linter to match the project's existing style.